packages/components/filters/filter-menu/src/trigger-button/trigger-button.tsx

```diff
@@ -0,0 +1,83 @@
+import { type ReactElement } from 'react';
+import IconButton from '@commercetools-uikit/icon-button';
+import { CaretDownIcon, CloseIcon } from '@commercetools-uikit/icons';
+import { Badge } from '../badge';
+import { Chip } from '../chip';
+import type { TAppliedFilterValue } from '../filter-menu';
+
+export type TFilterMenuTriggerButtonProps = {
+  /**
+   * formatted message to display the filter's name
+   */
+  label: string | ReactElement;
+  /**
+   * the values applied to this filter by the user
+   *
+   * NOTE / OPINIONS WELCOME IN PR COMMENTS:
+   * this will almost certainly be some sort of generic w/validation - all the `Chip` really
+   * needs is the selected option's `label`, but these options need to contain a lot more data
+   */
+  appliedFilterValues:
+    | TAppliedFilterValue
+    | TAppliedFilterValue[]
+    | undefined
+    | null;
+  /**
+   * whether or not the filter is disabled
+   */
+  isDisabled?: boolean;
+  /**
+   * indicates whether FilterMenu can be removed from the filtersList
+   */
+  isPersistent?: boolean;
+  /**
+   * controls whether `x` in Trigger Button is displayed - required if `isPersistent` is `false`
+   *
+   * NOTES / OPINIONS WELCOME IN PR COMMENTS:
+   * is the presence/absence of the function sufficient for display logic, or do we need a separate `show-` prop?
+   */
+  onRemoveFilter?: Function;
+};
+
+const TriggerButton = (props: TFilterMenuTriggerButtonProps) => {
+  return (
+    <button
```

Comment:
I just noticed that this will generate invalid markup since the `<IconButton>` below can not be inside another `<button>`-element.

Reply:
This is a really good point. Updated with a dirty hack.  I'm not sure how to achieve this design otherwise